shared/js/ui/templates/key-insights.js

```diff
@@ -52,6 +52,7 @@ const keyInsightsState = /** @type {const} */ ({
  */
 export function renderKeyInsight() {
     const model = this.model
+    const layout = this.model.featureSettings.primaryScreen?.layout || 'default'
```

Comment:
❓ If we're defaulting in the model, do we still need to default here?

Reply:
Good catch!